integration-tests/deployment/ccip/deploy_home_chain.go

```diff
@@ -52,6 +52,21 @@ const (
 	MaxDurationShouldTransmitAcceptedReport = 10 * time.Second
 )
 
+var (
+	// TODO: sort out why the mismatch here
+	//CCIPCapabilityId [32]byte = utils.MustHash(hexutil.Encode(MustABIEncode(`[{"type": "string"}, {"type": "string"}]`, CapabilityLabelledName, CapabilityVersion)))
+	CCIPCapabilityId = MustHashFromBytes(hexutil.MustDecode("0xe0da3c2b9005178f4731c9f40164f1933ad00bac9d6c13ad4ca1a8a763416380"))
+)
+
+func MustHashFromBytes(b []byte) [32]byte {
```

Comment:
```suggestion
func MustBytesToBytes32(b []byte) [32]byte {
```

Might be a better name since this doesn't actually hash anything